### PR TITLE
Privacy page revision

### LINKED ISSUE #758: 📱 Revise `Daily spending wallet` -> `Privacy` page
This issue will be part of a bigger milestone that focuses on revising the newly formed `Daily spending wallet` reference design we now have in the guide. 

The current privacy page in `Daily spending wallet` is more of a general overview of wallet privacy and isn't specific to a daily spending wallet. We should make the content of the page specific to daily spending wallets and add some UI designs illustrating the concepts. 

Some of the more general content may be more suited for a `Privacy` page in the `How it works` section which covers the technology / general considerations outside of specific contexts like a daily spending wallet. Though this is an issue for another time. 

Page: https://bitcoin.design/guide/daily-spending-wallet/privacy/

### PR DESCRIPTION
**Pull request by GBKS**
Addresses issue #758 about revising the privacy page of the daily spending wallet reference design. 
- Added an overview of different types of privacy
- Copy is more focused on user experience
- Added various images and replaced some existing images with screen mock-ups
- Re-ordered to put the user-facing options for IRL privacy at the top

There are really excellent articles out there that cover the technical aspects of privacy very well (like the [Bitcoin Wiki entry](https://en.bitcoin.it/wiki/Privacy)). We can just link out to those, and can keep this page pretty simple. 

I also came up with these 5 categories of privacy concerns:

1. Real-world privacy
2. Third-party services
3. Data transfer
4. Lightning network
5. Bitcoin network

Not sure if they hold up 100%, but I find that privacy discussions often get hard to follow because all the concerns are mixed up. Might be easier if we have a better framework?

Keeping this as a draft for now to have some discussion, but I am not totally comfortable with where it is right now.

Closes #758

### CODE REVIEW

File: guide/daily-spending-wallet/privacy.md [hunk 1]
```diff
@@ -244,23 +157,134 @@ An advantage here is the risk of an unauthorized person revealing their informat
...
+
+On the lightning network, a payment is only stored by the respective sender and receiver, and only as long as the channel in which the payment was made is open. However, opening and closing channels requires entries on the bitcoin blockchain, and those are also publicly stored forever. Additionally, lightning nodes are always online and usually directly tied to a single wallet, providing another data point. For a detailed analysis of privacy on lightning, see the [Security and privacy chapter](https://github.com/lnbook/lnbook/blob/develop/16_security_privacy_ln.asciidoc) in the Mastering the Lightning Network book.
+
+#### Be intentional when sharing static lightning identifiers
+
+Lightning node Ids, lightning addresses, and LNURL-Pay invoices (see [Payment request formats](https://bitcoin.design/guide/how-it-works/payment-request-formats/)) are examples of endpoints that can be used to generate many lightning invoices. While this is convenient for users, it is bad for privacy. For example, placing a lightning address on a website or social media profile makes it trivial to create a direct connection between the lightning node and the owner of the website or profile.
+
```
Comment (johnsBeharry) [inline guide/daily-spending-wallet/privacy.md:194]:
```suggestion

#### Be conscious about data leaks when generating invoices

When paying an invoice the origin is difficult / not possible to determine by the receiver. Generating and sharing invoices though will have encoded information about your node. In a daily spending wallet, it may not be practical to avoid this type of transaction but it may be something you may want to inform the wallet owner of during onboarding.
```

**Review by sbddesign — COMMENTED** [review]
Very nice!

I think the categories of real world, data transfer, 3rd party, lightning, and bitcoin make sense. 

Another way one might break this down is in terms of "threat models". How you weight each of those 5 categories would depend on the threat model hope you hope to defend against. Example: one person's threat model might be keeping casual snoops from discovering their wealth, another person's threat model might be evading authoritarianism. But that might be going into unneeded detail.

## My main takeaway

This is very good content that provides a very good overview of all privacy. **However:**

- There are times when I'm not sure how relevant it is to the DSW.
  - Example: discussion of coin control may not be relevant if this wallet moves everything on to lightning.
- There are other times when I am not sure which decision this DSW has made.
  - Example: "you may want...", "with this method...", etc.
  - What did *this wallet* do for its users?

## Suggested plan

- A) Refocus the content around which design choices this DSW has made, and then label some of this stuff as "alternate methods". I can provide very specific and targeted suggestions on how to do this.
- B) Let most of this content be a "How it Works" page and then create a smaller page just focusing on the privacy features of this DWS, and link out to this new "How it Works > Privacy" page quite liberally.

Personally, I'd vote for Plan B. Privacy is broad and will apply to many reference designs.

**Comment by GBKS** [comment]
@sbddesign good point. Thanks for bringing this back to the context of the daily spending wallet. Splitting it in 2 will clear things up a bit.

**Comment by GBKS** [comment]
The page is now split into two. The privacy page in the daily spending wallet is very focused on UI decisions for this particular use case. Other privacy info is split into a "Wallet privacy" page in "How it works". Not sure how much further I can push this working on it independently, so I'd like to open it up for review and feedback to move this PR forward.

File: guide/daily-spending-wallet/privacy.md [hunk 2]
```diff
@@ -45,222 +46,101 @@ This page should inform about what information is made public when sending or re
...
+Imagine this scenario. The user is in a public place, and they need to make a payment using their bitcoin wallet. They open the wallet on their phone, but they don’t feel comfortable having their address and balance information clearly visible to strangers who may be looking over their shoulder, persons lurking, or video surveillance. Hence, by giving users the ability to hide sensitive information in their wallet, but only when desired, they gain an added sense of physical privacy and security when using the app in public.
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/privacy.md:53]:
```suggestion
Imagine this scenario. The user is in a public place, and they need to make a payment using their bitcoin wallet. They open the wallet on their phone, but they don’t feel comfortable having their balance clearly visible to strangers who may be looking over their shoulder, persons lurking, or video surveillance. Giving users the ability to hide sensitive information in their wallet, but only when desired, they gain an added sense of physical privacy and security when using the app in public.
```

Wasn't sure what it meant by address

File: guide/daily-spending-wallet/privacy.md [hunk 3]
```diff
@@ -45,222 +46,101 @@ This page should inform about what information is made public when sending or re
...
-- Addresses - can be used to track on-chain transaction history
-- Invoices - can be used to track Lightning payment history
-- Private keys - can be used to access and transfer bitcoins
-
-It's more common for wallets to protect private keys, but not much is done for other sensitive information like the balance, addresses, and previous transactions. A few wallets like [Bitcoin Core](https://bitcoin.org/en/bitcoin-core/), [Wasabi](https://wasabiwallet.io/), [Muun](https://muun.com/), and others have made it work, though. Below are patterns and considerations for hiding and revealing sensitive information.
-
-### Quickly hiding balances
+#### Quickly hiding balances
 
 <div class="center" markdown="1">
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/privacy.md:57]: Why not just tap the balance to hide? Rather than add extra visuals with the eye icon. Could make it that hiding is a single tap and revealing is tap and hold (uses may accidently tap and reveal their balance when they don't want to.

File: guide/daily-spending-wallet/privacy.md [hunk 4]
```diff
@@ -45,222 +46,101 @@ This page should inform about what information is made public when sending or re
...
+   caption = "Users can enable PIN entry before the balance is revealed."
    width = 250
    height = 541
    layout = "float-right-desktop -background -shadow"
 %}
 
-With this method, it's as easy to reverse the hidden state as enabling it. This is good for convenience's sake. However, for protection against unauthorized access, perhaps the user should only be able to unhide their information if a PIN or password has been entered. This could therefore reaffirm the identity of the wallet owner for extra security.
-
-In this example the risk of an unauthorized person revealing their information is minimal due to the PIN required. However, it might not be convenient for the users to repeatedly put in their PIN when ever they want to reveal their information especially if they do so often.
+To prevent this problem, users can optionally enable an additional setting to require [PIN]({{ "/guide/daily-spending-wallet/security/#pin-protection" | relative_url }}) entry for unhiding sensitive information. This reaffirms the identity of the wallet owner for extra security. Making this optional allows users to find their own balance between convenience and security.
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/privacy.md:91]: May be worth outlining some examples of 'sensitive information'

File: guide/daily-spending-wallet/privacy.md [hunk 5]
```diff
@@ -45,222 +46,101 @@ This page should inform about what information is made public when sending or re
...
-
-<!--
-There is no perfect solution to guarantee 100% privacy that lasts forever because things can be revealed over time. Since transactions are forever public, even if all precautions are taken at the time of payment to ensure the highest degree of anonymity, future behaviors of the wallet owner or transacting parties can still degrade previously attained privacy. A high amount of diligence is necessary whenever users are transacting with bitcoin. The product should be able to guide, inform, and prevent them against privacy degrading actions.
--->
+On-chain [address re-use]({{ "/guide/how-it-works/wallet-privacy/#generate-a-new-address-for-each-on-chain-payment" | relative_url }}) is a common privacy problem due to the tracability across transactions. Addresses this wallet uses are swap addresses generated by a lightning service provider. This lifts the responsibility for address management out of the hands of the user. More on the [requesting page]({{ "/guide/daily-spending-wallet/requesting/" | relative_url }}).
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/privacy.md:141]: Can remove, this wallet only uses swap addresses which are controlled by the LWS.

File: guide/daily-spending-wallet/privacy.md [hunk 6]
```diff
@@ -45,222 +46,101 @@ This page should inform about what information is made public when sending or re
...
 
-[^1]: [Bitcoin.org - Protecting your privacy](https://bitcoin.org/en/protect-your-privacy)
-[^2]: [Top Seven Ways Your Identity Can Be Linked to Your Bitcoin Address](https://99bitcoins.com/know-more-top-seven-ways-your-identity-can-be-linked-to-your-bitcoin-address/)
-[^3]: [Bitcoin Wiki / Privacy](https://en.bitcoin.it/wiki/Privacy#Common-input-ownership_heuristic)
+Just like any other application, privacy can be impacted by the use of third-party services (analytics, currency exchange rates, etc) and data transfer itself, as well as any services provided by the application developer. For more on this, view the [wallet privacy]({{ "/guide/how-it-works/wallet-privacy/" | relative_url }}) page.
```
Comment (Bosch-0) [inline guide/daily-spending-wallet/privacy.md:144]: This page needs something on sharing invoices as they contain a lot of sensitive info. Maybe just something about how invoices should be shared over private communication channels. 

Using LSPs / LWS have some privacy considerations also, may want to add content around this

File: guide/how-it-works/wallet-privacy.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+Privacy is about the ability to control what information is available to others, and what they can do with it. On this page, we will provide a soft introduction to the topic, with a focus on what product builders and users can do to improve their payment privacy.
+
+#### Design with privacy in mind
```
Comment (Bosch-0) [inline guide/how-it-works/wallet-privacy.md:48]: This small title looks out of place, why not use a bigger one?

File: guide/how-it-works/wallet-privacy.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+-->
+
+## Real-world privacy
+
+Exposing wallet information to others around you can be a problem in terms of security and privacy, as well as personal comfort. Sensitive information includes:
+
+- Wallet Balance - shows how much is owned
+- Addresses - can be used to track on-chain transaction history
+- Invoices - can be used to track lightning payment history
+- Private keys - can be used to access and transfer bitcoins
```
Comment (Bosch-0) [inline guide/how-it-works/wallet-privacy.md:67]: Add recovery phrase to this list (designers would be more familiar with this term than private keys).

File: guide/how-it-works/wallet-privacy.md [hunk 9]
```diff
@@ -0,0 +1,206 @@
...
+
+It's more common for wallets to protect private keys, but not much is done for other sensitive information like the balance, addresses, and previous transactions. A few wallets like [Bitcoin Core](https://bitcoin.org/en/bitcoin-core/), [Wasabi](https://wasabiwallet.io/), [Muun](https://muun.com/), and others have made it work, though. For patterns and considerations for hiding and revealing sensitive information, see the [privacy page]({{ "/guide/daily-spending-wallet/privacy/" | relative_url }}) in the daily spending wallet reference design.
+
+## Third-party services
+
+Whether it is for retrieving exchange rates or using [lightning services]({{ "/guide/how-it-works/lightning-services/" | relative_url }}), interacting with centralized third-parties can leak user data. You may want to allow users to disable those features, choose the services they use, or use another method of anonymizing requests. It is also essential to only expose your users to services that you have vetted and deemed trustworthy.
+
+## Data transfer
```
Comment (Bosch-0) [inline guide/how-it-works/wallet-privacy.md:75]: Data transfer sounds odd, Networking is more accurate imo

**Review by Bosch-0 — APPROVED** [review]
Left comments, we should return to the wallet privacy page and really beef it up at a later stage

File: guide/daily-spending-wallet/privacy.md (hunk 3 above)
Comment (GBKS) [reply guide/daily-spending-wallet/privacy.md:57]: Both the balance and icon are tappable.

Tap and hold is a good addition. We'll need an indicator though that lets users know what to do. The first reaction is probably to assume single-tap to unhide because it only took a tap to hide.

File: guide/daily-spending-wallet/privacy.md (hunk 5 above)
Comment (GBKS) [reply guide/daily-spending-wallet/privacy.md:141]: That's what this paragraph explains.

File: guide/daily-spending-wallet/privacy.md (hunk 6 above)
Comment (GBKS) [reply guide/daily-spending-wallet/privacy.md:144]: What are specific examples that can be included on this page?

File: guide/how-it-works/wallet-privacy.md (hunk 7 above)
Comment (GBKS) [reply guide/how-it-works/wallet-privacy.md:48]: Had that initially, but that looked out of place to me. Also tried removing it, but felt like the text below needs a header.

File: guide/daily-spending-wallet/privacy.md (hunk 1 above)
Comment (GBKS) [reply guide/daily-spending-wallet/privacy.md]: Can you explain this a bit? I don't quite get it.

File: guide/how-it-works/wallet-privacy.md (hunk 8 above)
Comment (GBKS) [reply guide/how-it-works/wallet-privacy.md]: Added. Also added transaction history.

**Comment by GBKS** [comment]
@Bosch-0 I think after this milestone, we need to get away from full-page overhauls when possible. Each individual privacy thing deserves its own discussion and issue/PR to get through. Lumping it all together creates these massive PRs that are rally hard to wrap up.

File: guide/how-it-works/wallet-privacy.md [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+Privacy is about the ability to control what information is available to others, and what they can do with it. On this page, we will provide a soft introduction to the topic, with a focus on what product builders and users can do to improve their payment privacy.
```
Comment (sbddesign) [inline guide/how-it-works/wallet-privacy.md:41]:
```suggestion
Privacy is about the ability to control what information is available to others and what they can do with it. On this page, we will provide a soft introduction to the topic, with a focus on what product builders and users can do to improve their payment privacy.
```

File: guide/how-it-works/wallet-privacy.md [hunk 11]
```diff
@@ -0,0 +1,198 @@
...
+- Invoices - can be used to track lightning payment history
+- Recovery phrase - can be used to access and transfer bitcoins
+
+It's more common for wallets to protect private keys, but not much is done for other sensitive information like the balance, addresses, and previous transactions. A few wallets like [Bitcoin Core](https://bitcoin.org/en/bitcoin-core/), [Wasabi](https://wasabiwallet.io/), [Muun](https://muun.com/), and others have made it work, though. For patterns and considerations for hiding and revealing sensitive information, see the [privacy page]({{ "/guide/daily-spending-wallet/privacy/" | relative_url }}) in the daily spending wallet reference design.
+
+## Third-party services
+
+Whether it is for retrieving exchange rates or using [lightning services]({{ "/guide/how-it-works/lightning-services/" | relative_url }}), interacting with centralized third-parties can leak user data. You may want to allow users to disable those features, choose the services they use, or use another method of anonymizing requests. It is also essential to only expose your users to services that you have vetted and deemed trustworthy.
```
Comment (sbddesign) [inline guide/how-it-works/wallet-privacy.md:65]:
```suggestion
Whether it is for retrieving exchange rates or using [lightning services]({{ "/guide/how-it-works/lightning-services/" | relative_url }}), interacting with centralized third parties can leak user data. You may want to allow users to disable those features, choose the services they use, or use another method of anonymizing requests. It is also essential to only expose your users to services that you have vetted and deemed trustworthy.
```

File: guide/how-it-works/wallet-privacy.md [hunk 12]
```diff
@@ -0,0 +1,198 @@
...
+
+> Each bitcoin transaction contains at least one input and at least one output. This means that once a single address is known, there is a trail to follow the bitcoin.
+>
+> <cite>As documented by <a href="https://docs.wasabiwallet.io/FAQ/FAQ-GeneralBitcoinPrivacy.html#how-is-bitcoin-bad-in-terms-of-privacy">Wasabi Wallet</a></cite>
+
+There are [many ways](https://99bitcoins.com/know-more-top-seven-ways-your-identity-can-be-linked-to-your-bitcoin-address/) your identity might get connected to your wallet and payments, so keeping bitcoin payments private takes diligent work, but is not impossible. Let’s explore some practices that help preserve privacy of bitcoin payments.
```
Comment (sbddesign) [inline guide/how-it-works/wallet-privacy.md:119]:
```suggestion
There are [many ways](https://99bitcoins.com/know-more-top-seven-ways-your-identity-can-be-linked-to-your-bitcoin-address/) your identity might get connected to your wallet and payments, so keeping bitcoin payments private takes diligent work but is not impossible. Let’s explore some practices that help preserve the privacy of bitcoin payments.
```

File: guide/how-it-works/wallet-privacy.md [hunk 13]
```diff
@@ -0,0 +1,198 @@
...
+   retina = "/assets/images/guide/how-it-works/wallet-privacy/activity-labeling@2x.png"
+   modelImage = "/assets/images/guide/how-it-works/wallet-privacy/activity-labeling-big.png"
+   alt-text = "Screen showing a transaction list with contacts and descriptions."
+   caption = "A well-annotated transaction list can simplify privacy management."
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+There are multiple benefits to users adding [contacts]({{ "/guide/daily-spending-wallet/contacts/" | relative_url }}) and [additional details]({{ "/guide/daily-spending-wallet/activity/" | relative_url }}) to transactions. It helps remember what payments were for, and also enables preventative measures for preserving privacy. In future transactions, it becomes easier to send coins ([UTXOs]({{ "/guide/glossary/#unspent-transaction-output-utxo" | relative_url }})) that do not reveal parts of the users transaction history that they are uncomfortable being public knowledge. This is often referred to as [coin control]({{ "/guide/glossary/#coin-control" | relative_url }}).
```
Comment (sbddesign) [inline guide/how-it-works/wallet-privacy.md:169]:
```suggestion
There are multiple benefits to users adding [contacts]({{ "/guide/daily-spending-wallet/contacts/" | relative_url }}) and [additional details]({{ "/guide/daily-spending-wallet/activity/" | relative_url }}) to transactions. It helps remember what payments were for and also enables preventative measures for preserving privacy. In future transactions, it becomes easier to send coins ([UTXOs]({{ "/guide/glossary/#unspent-transaction-output-utxo" | relative_url }})) that do not reveal parts of the user’s transaction history that they are uncomfortable being public knowledge. This is often referred to as [coin control]({{ "/guide/glossary/#coin-control" | relative_url }}).
```

File: guide/daily-spending-wallet/privacy.md [hunk 14]
```diff
@@ -45,222 +41,103 @@ This page should inform about what information is made public when sending or re
...
+   modalImage = "/assets/images/guide/daily-spending-wallet/privacy/hidden-balance-reveal-timer-big.png"
+   alt-text = "A mobile interface where sensitive information is hidden by default"
+   caption = "A timer counts to zero before automatic balance reveal."
    width = 250
    height = 541
    layout = "float-right-desktop -background -shadow"
 %}
 
-Having the show/hide button right on the main screen makes things quite obvious for someone who has access to a user's device to press unhide. A solution would be to move the hide toggle away from the home screen and into the [app settings](https://medium.com/@olanrewajusodiq64/the-ui-ux-of-hide-balance-designing-to-improve-asset-security-e4b20668f315). This way, if someone has access to their device and opens the app, they may not immediately know how to reveal the balance, transaction, or addresses as it is not made obvious as the previous solutions.
+Another solution is to invoke the wallet’s hidden state as a default when the app is opened, to protect against prying eyes during initial display. The pre-hidden state can be unveiled after a tap, PIN entry, or perhaps a short 5-second timer.
```
Comment (sbddesign) [inline guide/daily-spending-wallet/privacy.md:130]:
```suggestion
Another solution is to invoke the wallet’s hidden state as a default when the app is opened to protect against prying eyes during initial display. The pre-hidden state can be unveiled after a tap, PIN entry, or perhaps a short 5-second timer.
```

File: guide/daily-spending-wallet/privacy.md [hunk 15]
```diff
@@ -45,222 +41,103 @@ This page should inform about what information is made public when sending or re
...
-
-<!--
-There is no perfect solution to guarantee 100% privacy that lasts forever because things can be revealed over time. Since transactions are forever public, even if all precautions are taken at the time of payment to ensure the highest degree of anonymity, future behaviors of the wallet owner or transacting parties can still degrade previously attained privacy. A high amount of diligence is necessary whenever users are transacting with bitcoin. The product should be able to guide, inform, and prevent them against privacy degrading actions.
--->
+On-chain [address re-use]({{ "/guide/how-it-works/wallet-privacy/#generate-a-new-address-for-each-on-chain-payment" | relative_url }}) is a common privacy problem due to the tracability across transactions. Addresses this wallet uses are swap addresses generated by a lightning service provider. This lifts the responsibility for address management out of the hands of the user. More on the [requesting page]({{ "/guide/daily-spending-wallet/requesting/" | relative_url }}).
```
Comment (sbddesign) [inline guide/daily-spending-wallet/privacy.md:138]:
```suggestion
On-chain [address re-use]({{ "/guide/how-it-works/wallet-privacy/#generate-a-new-address-for-each-on-chain-payment" | relative_url }}) is a common privacy problem due to the traceability across transactions. Addresses this wallet uses are swap addresses generated by a lightning service provider. This lifts the responsibility for address management out of the hands of the user. More on the [requesting page]({{ "/guide/daily-spending-wallet/requesting/" | relative_url }}).
```

**Review by sbddesign — CHANGES_REQUESTED** [review]
Looking good. I like this as 2 separate pages. Left spelling and grammar edits since I think this one is pretty close to ready.

File: guide/how-it-works/wallet-privacy.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (Bosch-0) [reply guide/how-it-works/wallet-privacy.md:41]: Agree

**Review by Bosch-0 — APPROVED** [review]

File: guide/how-it-works/wallet-privacy.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (GBKS) [reply guide/how-it-works/wallet-privacy.md:41]: Do you mean in the context of this page or in general?

Don't privacy policies usually also include descriptions on how data is used? Here's a screenshot from an App Store listing whose privacy section describes both what data is collected and how it is used ([App store guidelines](https://developer.apple.com/app-store/app-privacy-details/#data-type-usage)):
app-store-app-privacy

**Comment by GBKS** [comment]
@sbddesign @Bosch-0 addresses the feedback. Just have that one point around whether privacy includes confidentiality.

File: guide/how-it-works/wallet-privacy.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (Bosch-0) [reply guide/how-it-works/wallet-privacy.md]: When accepting that policy though you are choosing to reveal it like Stephen said, once they have it they can do what they want with it

[the remark this answers is not in the document]
Comment (sbddesign) [reply guide/how-it-works/wallet-privacy.md]: > Do you mean in the context of this page or in general?

Both.

In general: consulted with 3 dictionaries and I can't find any mention of privacy meaning "to control what other people do with your information". The word typically refers to a state of concealment or the right to be in a state of concealment.

In the context of this page: privacy policies are a non-sequitur. They are not applicable because we are talking mostly about real-world privacy and bitcoin network privacy, not information we share with companies. But even if we do want to talk about company privacy policies, I'd argue that the privacy policy does not give us any control. A company can have a very strict sounding privacy policy and still have the ability to abuse the policy. If nobody figures out, the company gets away with it. If somebody does figure out, there may be legal action, but that doesn't change the fact that the abuse _already happened_.

In the screenshot you shared, that screen does not give the user any control over what TikTok does with their data. It only gives the user control over what data they choose to share.

I'm reading through both of these privacy pages and I'm having trouble finding an example of a place where it discusses revealing information to somebody and then having an ability to control what that person does with the information.

**Review by Bosch-0 — APPROVED** [review]

File: guide/how-it-works/wallet-privacy.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (GBKS) [reply guide/how-it-works/wallet-privacy.md]: Sure, let's go with your recommendation.

**Review by Bosch-0 — APPROVED** [review]

**Review by sbddesign — APPROVED** [review]
